src/preset.ts

```diff
@@ -338,11 +377,24 @@ interface Toolset {
 /**
  * Should NOT cache anything. Need to make a copy if any fields need to be changed.
  */
+
+// the strings don't match the spec???
```

Comment:
What strings don't match what spec?

Reply:
Stale comment. Spec says "Custom Workflow Preset" and we're generating "" for display name and description. This would be something specific to all presets "add new" wizard. I'll open a work item and no need to fix now before the string freeze.